pkg/model/components/apiserver.go

```diff
@@ -296,7 +296,7 @@ func (b *KubeAPIServerOptionsBuilder) BuildOptions(o interface{}) error {
 		c.AnonymousAuth = fi.Bool(false)
 	}
 
-	if b.IsKubernetesGTE("1.18") {
+	if b.IsKubernetesGTE("1.17") {
```

Comment:
You'll want to change this on master branch as well. How about first landing to master and then cherrypicking?

Reply:
Fair point, that's an easy follow up.

Reply:
Oops - will send PR to master now :-)